<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDlg</class>
 <widget class="QDialog" name="PasswordDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>227</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>228</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="m_LblPassword1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_EnterPasswdLineEdit1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_LblPassword2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LblPassword3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>57</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_EnterPasswdLineEdit2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_EnterPasswdLineEdit3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_LeftButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_RightButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_msgLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
